ros-kinetic/qt_gui_cpp/qt_gui_cpp-0.3.11.ebuild

```diff
@@ -25,6 +25,8 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig
 	dev-qt/qtcore:5
 	dev-qt/qtcore:5
+	dev-qt/widgets:5
```

Comment:
@awesomebytes this is not an ebuild... Fortunately, it pulled things in correctly, but this should definitely be `dev-qt/qtwidgets:5`.